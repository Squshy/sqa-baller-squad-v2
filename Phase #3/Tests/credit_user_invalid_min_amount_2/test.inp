LOGIN
MR.RUNESCAPEMAN
buyinggf20k
ADDCREDIT
0.00
logout
exit