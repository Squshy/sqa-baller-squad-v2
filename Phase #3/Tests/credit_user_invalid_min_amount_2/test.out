Input credit amount: 0.00
--------------------------------------------
You must enter a minimum of $0.01 credits.
============================================